source/crud/examples/java/src/main/java/examples/MongoCollectionUsageExample.java

```diff
@@ -126,11 +126,23 @@ public static void usageExample(MongoCollection<Document> col) {
         col.updateOne(new Document("x", 1), new Document("$set", new Document("x", 2)),
                       new UpdateOptions().upsert(true));
 
+        // using a pipeline for the update parameter
+        col.updateOne(new Document("x", 1), asList(new Document("$set", new Document("x", 2))));
```

Comment:
AFAIK, the `crud/examples` directory is obsolete. I'd propose just deleting the entire directory (and any references to it) in a subsequent PR.

Most drivers now coordinate with the docs team to maintain CRUD examples in our test suites, which are then published in the MongoDB manual. We should rely on that going forward. If the docs team has not already done so, I'd suggest reaching out to ensure they have a tracking ticket to request examples for update pipelines.

Reply:
I have reverted all changes to the examples files.